src/lib.cairo

```diff
@@ -4,8 +4,6 @@ pub mod utils {
     pub mod bit_shifts;
     pub mod merkle_tree;
     pub mod numeric;
-
-    #[cfg(target: 'test')]
```

Comment:
There’s a reason for that, hex helpers are not supposed to be used outside of tests, use kore efficient representation

Reply:
@m-kus, what do you mean by kore efficient representation?